src/commands.js

```diff
@@ -706,6 +706,19 @@ export function goToNextCell(direction) {
   };
 }
 
+// :: (EditorState, ?(tr: Transaction)) → bool
+// Selects the current cell the cursor is in
+export function selectCurrentCell(state, dispatch) {
+  const currentCell = selectionCell(state);
+  if (!currentCell || !dispatch) {
+    return false;
+  }
+  if (dispatch) {
```

Comment:
nit: Don't need this if statement now

Reply:
Yes of course... It really is the end of the day over here :)